core/src/com/goonsquad/galactictd/screens/ScoreScreen.java

```diff
@@ -76,7 +82,6 @@ private void generateFont() {
 
     @Override
     public void render(float delta) {
-        Gdx.app.debug(TAG, "render() called");
```

Comment:
I just figured this wouldn't be quite useful since it would be called every frame. If you think it is, I'm fine with leaving it in.